sinch/core/clients/sinch_client_configuration.py

```diff
@@ -32,6 +32,9 @@ def __init__(
         self.auth_origin = "auth.sinch.com"
         self.numbers_origin = "numbers.api.sinch.com"
         self.verification_origin = "verification.api.sinch.com"
+        self.voice_applications_origin = "callingapi.sinch.com"
```

Comment:
I think it's not used, and overrieded below, but it would be a dot after `calling`
```suggestion
        self.voice_applications_origin = "calling.api.sinch.com"
```

Reply:
https://developers.sinch.com/docs/voice/api-reference/voice/tag/Applications/
I think there is a separate endpoint for `Applications`:
`url = "https://callingapi.sinch.com/v1/configuration/numbers"`